package.json

```diff
@@ -40,6 +40,7 @@
     "istanbul-lib-coverage": "^3.2.0",
     "istanbul-lib-report": "^3.0.1",
     "istanbul-reports": "^3.1.6",
+    "monocart-coverage-reports": "^2.8.3",
```

Comment:
similar to what is being discussed [here](https://github.com/bcoe/c8/pull/518) let's make monocart a `peerDependency` and update our docs to include:

```
npm i monocart-coverage-reports --save-dev
```

I think this is the right call since monocart is fairly large:

```
npm notice package size:  253.8 kB                                
npm notice unpacked size: 905.1 kB   
```

And I don't think it adds that much of a hinderance to people wanting to use it.

Reply:
For reference: https://flaviocopes.com/npm-peer-dependencies/

Reply:
I moved it to peerDependencies, it good a suggestion. Due to the new V8's report having a very complex UI, it take a large size.